en/docs/connectors/catalog/messaging/asb/actions.md

```
@@ -198,7 +198,7 @@ Receive messages from queues and subscriptions, settle messages, and manage lock
 | Field | Type | Default | Description |
 |-------|------|---------|-------------|
 | `connectionString` | `string` | Required | The Azure Service Bus connection string. |
-| `entityConfig` | `QueueConfig|TopicSubsConfig` | Required | Entity configuration — either `{queueName: "..."}` for queues or `{topicName: "...", subscriptionName: "..."}` for topic subscriptions. |
+| `entityConfig` | `QueueConfig|TopicSubsConfig` | Required | Entity configuration: either `{queueName: "..."}` for queues or `{topicName: "...", subscriptionName: "..."}` for topic subscriptions. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)